.env.example

```diff
@@ -43,4 +43,8 @@ CONFIRM_EMAIL=false
 DISABLE_NON_BUSINESS_EMAIL_SIGNUP=false
 
 # Mixpanel
-NEXT_PUBLIC_MIXPANEL_TOKEN=
\ No newline at end of file
+NEXT_PUBLIC_MIXPANEL_TOKEN=
+
+# Enable Auth providers (comma separated)
+# Supported providers: github, google, saml, email
+AUTH_PROVIDERS=github,google,saml,email
```

Comment:
We just want github,email in the end as default

Reply:
And if not defined default to github,email

Reply:
Ok, noted.